aten/src/THCUNN/generic/SpatialConvolutionMM.cu

```diff
@@ -81,6 +81,19 @@ static inline void THNN_(SpatialConvolutionMM_shapeCheck)(
   }
 }
 
+static THCTensor* THNN_(newViewWeightMM2d)(THCState *state, THCTensor *weight) {
```

Comment:
Ideally, the name of this function would make it clearer that it steals ownership of the input `weight`